cubic-server/Dimension.hpp

```diff
@@ -23,7 +22,6 @@ class Entity;
 class Dimension {
 private:
     struct ChunkRequest {
```

Comment:
This could be change for something like this I think
```suggestion
    typedef ChunkRequest std::vector<std::weak_ptr<Player>> players;
```